Doc/library/os.rst

```diff
@@ -1179,9 +1183,28 @@ as internal buffering of data.
    Parameters to the :func:`lseek` function. Their values are 0, 1, and 2,
    respectively.
 
+
+.. data:: SEEK_HOLE
+          SEEK_DATA
+
+   Parameters to the :func:`lseek` function and the :meth:`~io.IOBase.seek`
+   method on file objects, for seeking file data and holes on sparsely
+   allocated files.
```

Comment:
I'm just noticing that this doesn't say what *pos* means if `SEEK_HOLE` or `SEEK_DATA` is passed as *how*.

Reply:
Hm, how about this:

```diff
diff --git a/Doc/library/os.rst b/Doc/library/os.rst
index 9bc5fd4da6..00a73277d3 100644
--- a/Doc/library/os.rst
+++ b/Doc/library/os.rst
@@ -1192,10 +1192,12 @@ as internal buffering of data.
    allocated files.
 
    :data:`!SEEK_DATA`
-      Adjust the file offset to the next location in the file containing data.
+      Adjust the file offset to the next location in the file,
+      greater than or equal to the seek position, containing data.
 
    :data:`!SEEK_HOLE`
-      Adjust the file offset to the next hole in the file.
+      Adjust the file offset to the next hole in the file,
+      greater than or equal to the seek position.
       A hole is defined as a sequence of zeros.
 
    .. note::
```

Reply:
Or can we assume it is understood by the implied meaning in the start of the sentence: "adjust the file offset to the next location ..."?

Reply:
What I'm trying to say is: "the next location in the file" and "greater than or equal to the seek position" is two ways of saying the same thing.

Reply:
Right, that's different from my interpretation. If you are ok with my recent suggestion, I'll apply that.

Reply:
Wording adjustment to remove the comma splice: "Adjust the file offset to the next location in the file containing data, greater than or equal to the seek position." or "Adjust the file offset to the next location in the file that contains data and is greater than or equal to the seek position."

Reply:
Thanks; that reads nice.

Reply:
I think "in the file" is superfluous. How about this:

_"Adjust the file offset to the next location relative to the seek position, containing data"_

Reply:
@pitrou: are you happy with e580a687ceba12d26fb4e6b0848fe8ed5a76a7e6?

Reply:
@erlend-aasland Yes, thanks.